docs/source/create-ai-pipeline.md

```diff
@@ -92,6 +98,12 @@ where `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `OBJECT_STORAGE_ENDPOINT_URL
 <img alt="Insert inputs in Elyra Runtime" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/InsertInputsElyraRuntime.png">
 </div>
 
+⚠️ Please note, if you're using a secure bucket, to avoid exposing your Cloud Object Storage credentials, you can use the `Cloud Object Storage Credentials Secret` which is a Kubernetes secret that’s defined in the Kubeflow namespace, containing the Cloud Object Storage username and password. This secret must exist on the Kubernetes cluster hosting your pipeline runtime in order to successfully execute pipelines.
+
+Refer this [template](https://elyra.readthedocs.io/en/stable/user_guide/runtime-conf.html#cloud-object-storage-credentials-secret-cos-secret) for an example of how to define your secret on the Kubernetes cluster hosting your runtime.
+
+We have a secret defined on the Operate First Kubeflow Namespace called `opf-datacatalog-bucket` available for the `opf-datacatalog` bucket which can be used for the `Cloud Object Storage Credentials Secret` field if you are using the `opf-datactalog` bucket here.
```

Comment:
will users know if they are/should be using the `opf-datacatalog` bucket? is this the bucket for those who don't have buckets (for lack of better explanation :) )?

also-- the last *opf-datacatalog is missing an "a"

Reply:
good question. As we discussed here https://github.com/AICoE/elyra-aidevsecops-tutorial/issues/216, they should be offered a public bucket to run their demos with, and I believe we decided to go with the `opf-datacatalog`. If there is another bucket we wish to offer credentials for to the user, we should create a secret for that and link it here.

Reply:
fixed the typo.